filters/filters.txt

```diff
@@ -5092,5 +5092,6 @@ xfreehd.com##.nvheader > span:has-text(Advertisement)
 ! https://github.com/uBlockOrigin/uAssets/issues/1550
 ||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$script,redirect=googlesyndication.com/adsbygoogle.js,domain=downloadfullfree.com
 
-! https://github.com/uBlockOrigin/uAssets/issues/1551
```

Comment:
@okiehsch i did not delited your commit but we have committed the filter.txt file at the same time...